### DISCUSSION TITLE
Image extracted but not displayed

### DISCUSSION THREAD

**Discussion by leinaht** [aditya-K2/gomp#56] [Q&A] [answered]
I'm seeing images extracted to my cache folder but nothing displayed in the relevant window. Messing with the padding and resizing didn't get me anywhere - what could I be missing? Gomp is running on a headless Debian bookworm server accessed via ssh.

**Comment by akurdunkar** [comment, marked as answer]
Hi, gomp uses the x child windows to render images. As you're running a headless setup, there is no x server that is running, hence no image is displayed.
